bootstrap/kubeadm/types/v1beta1/conversion_test.go

```diff
@@ -37,26 +40,43 @@ func TestFuzzyConversion(t *testing.T) {
 		Spoke:  &ClusterConfiguration{},
 		// NOTE: Kubeadm types does not have ObjectMeta, so we are required to skip data annotation cleanup in the spoke-hub-spoke round trip test.
 		SkipSpokeAnnotationCleanup: true,
+		FuzzerFuncs:                []fuzzer.FuzzerFuncs{fuzzFuncs},
 	}))
 	t.Run("for ClusterStatus", utilconversion.FuzzTestFunc(utilconversion.FuzzTestFuncInput{
 		Scheme: scheme,
 		Hub:    &v1alpha4.ClusterStatus{},
 		Spoke:  &ClusterStatus{},
 		// NOTE: Kubeadm types does not have ObjectMeta, so we are required to skip data annotation cleanup in the spoke-hub-spoke round trip test.
 		SkipSpokeAnnotationCleanup: true,
+		FuzzerFuncs:                []fuzzer.FuzzerFuncs{fuzzFuncs},
```

Comment:
nit: 
FuzzerFunc are not strictly required here, given that `ClusterStatus` does not contains DNS (same for `InitConfiguration` and `JoinConfiguration`). However, they should not create a problema and make the code ready for future changes, so I'm also ok to leave the

Reply:
Ok!